client/src/components/Result/index.jsx

```diff
@@ -1,22 +1,85 @@
-import { BsFillCheckCircleFill } from 'react-icons/bs';
+import { BsFillCheckCircleFill, BsFillExclamationCircleFill, BsFillExclamationTriangleFill } from 'react-icons/bs'; //get logos from here
 
-const Result = function Result() {
-  return (
-    <div
-      style={{
+const Result = function Result(status, title, intro) {
+  let comp;
+
+  if ( { status } === "success" ) {
+    comp = (
+      <div style={{
         width: '100%',
         height: '100%',
         display: 'flex',
         justifyContent: 'center',
         alignItems: 'center',
       }}
-    >
-      <BsFillCheckCircleFill
-        style={{
-          fontSize: '72px',
-          color: '#52c41a',
-        }}
-      />
+      >
+        <BsFillCheckCircleFill
+          style={{
+            fontSize: '72px',
+            color: '#52c41a',
+          }}
+        />
+      </div>
+    )
+  } else if ( { status } === "error" ) {
+    comp = ( 
+      <div style={{
+        width: '100%',
+        height: '100%',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+      >
+        <BsFillExclamationTriangleFill
+          style={{
+            fontSize: '72px',
+            color: '#52c41a',
+          }}
+        />
+      </div>
+    )
+  } else if ( { status } === "info" ) {
+    comp = (
+      <div style={{
+        width: '100%',
+        height: '100%',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+      >
+        <BsFillExclamationCircleFill
+          style={{
+            fontSize: '72px',
+            color: '#52c41a',
+          }}
+        />
+      </div>
+    )
+  } else if ( { status } === "warning" ) {
+    comp = (
+      <div style={{
+        width: '100%',
+        height: '100%',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+      >
+        <BsFillExclamationCircleFill
+          style={{
+            fontSize: '72px',
+            color: '#52c41a',
+          }}
+        />
+      </div>
+    )
+  }
+  
+  return (
+    <div>
+      { status, title, intro }
```

Comment:
You are simply displaying the props.

Reply:
Yeah this was from me just trying to figure out and understand the syntax. This was just a testing measure and will be changed.

Reply:
You have marked the pull request as ready to review, and I am worrying, since this is not something that you still need to figuring out after four weeks since initial task assignment.